data/json/recipes/recipe_food.json

```diff
@@ -8666,5 +8666,41 @@
     "qualities": [ { "id": "CUT", "level": 1 }, { "id": "COOK", "level": 1 } ],
     "tools": [ [ [ "surface_heat", 8, "LIST" ] ] ],
     "components": [ [ [ "cactus_pad", 3 ] ] ]
+  },
+  {
+    "type": "recipe",
+    "result": "homemade_burrito",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_MEAT",
+    "skill_used": "cooking",
+    "difficulty": 4,
+    "charges": 3,
+    "time": "30 m",
+    "autolearn": true,
+    "qualities": [ { "id": "CUT", "level": 1 }, { "id": "COOK", "level": 2 } ],
+    "tools": [ [ [ "surface_heat", 10, "LIST" ] ] ],
+    "components": [
+      [ [ "meat", 1 ], [ "dry_meat", 1 ], [ "can_chicken", 1 ], [ "meat_smoked", 1 ] ],
+      [ [ "tortilla_corn", 3 ] ],
+      [
+        [ "dry_beans", 1 ],
+        [ "raw_beans", 1 ],
+        [ "dry_lentils", 1 ],
+        [ "raw_lentils", 1 ],
+        [ "dry_rice", 1 ],
+        [ "can_beans", 1 ],
+        [ "beans_cooked", 1 ]
+      ],
+      [
+        [ "lettuce", 2 ],
+        [ "irradiated_lettuce", 2 ],
+        [ "dandelion_cooked", 2 ],
+        [ "veggy_wild_cooked", 2 ],
+        [ "irradiated_cabbage", 2 ],
+        [ "cabbage", 2 ]
+      ],
+      [ [ "tomato", 1 ], [ "irradiated_tomato", 1 ], [ "can_tomato", 1 ] ],
```

Comment:
you could add red_sauce here, and maybe some other sauces to substitute for tomato.

Reply:
It's not about sauce, it's just like... diced tomato chunks mixed with the vegetables [like this](https://www.recipetineats.com/wp-content/uploads/2017/06/Burritos-DIY-650x1099.jpg) see the tomato chunks? unless the red sauce in the game is one of those that is very chunky (those exist right?)